abi/src/specs.rs

```diff
@@ -169,7 +169,7 @@ impl ContractSpecBuilder<Valid> {
 
 impl ContractSpec {
     /// Creates a new contract specification.
-    pub fn new(name: <MetaForm as Form>::String) -> ContractSpecBuilder {
+    pub fn builder(name: <MetaForm as Form>::String) -> ContractSpecBuilder {
```

Comment:
I am against naming this `builder` even though this would fix a `clippy` lint.
However, we should sometimes question certain clippy lints and also we could come up with better names.
For example `builder` is not suitable but maybe `build` was, but then again people could be confused since we can also `build` (aka compile) smart contracts.

Reply:
Let's allow to name this method `new` for now please.